### DISCUSSION TITLE
Manual PR Body Editing should be possible - Renovate should not revert the change

### DISCUSSION THREAD

**Discussion by DennisX1** [renovatebot/renovate#28930] [Suggest an Idea]
### Tell us more.

We define the test configuration for the PR in our PR Body.

If we adjust the body, Renovate will revert the change, which is a problem.

On the one hand it happens that Jenkins is fetching the wrong test configuration (because Renovate reverts the change) , and on the other hand it makes the documentation of the executed tests more complex.

It would be great if there would be a possibility to tell Renovate that it should not edit the PR Body if it is manually changed.

Maybe in the Renovate configuration itself or with some special syntax that tells Renovate ''Everything that is written between these signs , should not be reverted"

There was an open issue , which also discussed this idea (already back in 2022) , but it was unfortunately not realised

**Comment by rarkins** [comment]
Which platform?

**Reply by rarkins** [reply]
GitHub, GitLab, etc

**Reply by DennisX1** [reply]
GitHub

**Reply by rarkins** [reply]
I think for platforms like GitHub which support html hidden comments, we could add a start and stop delimiter for our content and then let users or other bots edit anything they want before or after. We want to avoid users modifying Renovate's own content as otherwise we can't update a PR after we've created it.

**Reply by DennisX1** [reply]
That would be great - as long as I can put sth in the PR body and Renovate is not reverting this I will be very happy :)

Thanks already!

**Reply by DennisX1** [reply]
I just thought about one problem with this approach: We define our test config in the footer (which is defined in the Renovate config)
So if I want to adjust it, it would be required to delete the footer and create a new one before or after the delimiter.
However, Renovate would then add the footer again in the delimiter, which would mean we have then 2 footers.

Maybe its possible to add an optional parameter in the Renovate config, which allows to add additional things to the PR body, which Renovate will not touch

Or (but this is special to my case) add an parameter to the footer to tell Renovate, that it should not edit the footer

**Reply by rarkins** [reply]
I created this issue: https://github.com/renovatebot/renovate/issues/28933

In short: do not touch Renovate's content, but you can add anything you want before or after Renovate's content and it will be retained, even if Renovate needs to update its own content.
